### ISSUE TITLE
`zpool status -L` could show physpath

### ISSUE THREAD

**Issue by robn** [openzfsonwindows/openzfs#201]
`zpool status -L` normally shows you the /dev node after all symlinks (like /dev/by-id/xxx) are resolved. Obviously on Windows the /dev nodes aren't real so there's nothing more to resolve.

It does seem that there's a mapping though, shown as `physpath` in the debug output when importaing a pool.

```
setting path here '/dev/physicaldrive3'
setting physpath here '#131072#999645101568#\\?\scsi#disk&ven_lsi&prod_raid_5#6_sas_6g#5&2d643ee&0&010000#{53f56307-b6bf-11d0-94f2-00a0c91efb8b}'
```

Showing some or all of that somewhere would be great, and since `zpool status -L` isn't useful right now, that might make it work closer to the original in spirit.

(there's a good argument that this might be better as a vdev property when that lands too)

```
C:\>zpool version
zfswin-2.1.6rc2
zfs-kmod-zfswin-2.1.6rc2
```

**Comment by lundman** [comment]
Yeah, since Windows does not handle the EFI style partitions that ZFS uses, we encode
`#offset#length#/path` in `vdev_physpath` as it is one (of very few) pieces of information passed to kernel using import time.

Of course, we could trim the ## part of phys path with `-L` usage, but it does refer to just a slice.

Ultimately, we should eventually get a separate piece of code/driver/util that does the EFI label, and creates the HarddiskXPartitionY style symlinks that Windows expects, and we'd only have the phys path in there. One day..
